pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -111,17 +110,27 @@ contract TimelockAuthorizer is IAuthorizer, IAuthentication, ReentrancyGuard {
     IAuthorizerAdaptor private immutable _authorizerAdaptor;
     uint256 private immutable _rootTransferDelay;
 
+    // Authorizer permissions
     address private _root;
     address private _pendingRoot;
-    ScheduledExecution[] private _scheduledExecutions;
+    mapping(uint256 => mapping(address => bool)) private _isExecutor;
```

Comment:
Maybe add a comment describing what the outer / inner mappings represent?
For simple mappings the variable name is usually enough, but in this case a comment would help.

Reply:
Something like:
// scheduledExecutionId => (executorAddress => true/false)